src/main/java/uk/gov/hmcts/reform/waworkflowapi/controllers/startworkflow/CreateTaskRequest.java

```diff
@@ -22,6 +31,10 @@ public Transition getTransition() {
         return transition;
     }
 
+    public String getDueDate() {
+        return dueDate;
+    }
+
     @Override
     public boolean equals(Object object) {
```

Comment:
suggestion: Even if this could be perfectly fine I think it might lead to difficult issues to spot. See this, please: https://www.ideyatech.com/blog/2011/04/13/effective-java-equals-and-hashcode/

It's always a good practice to adhere to those 5 rules. 

suggestion: Lombok could handle this more easily for us https://projectlombok.org/features/EqualsAndHashCode

Reply:
We decided not to use Lombok the other day I did say at the time that it would avoid the code coming up in code coverage but after talking about it the decision was not to use Lombok.

The equals method is just the one generated by intellij which is what we decided to use